missing_strings.sh

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
this script has little sense, it's fairly easy to check against english.json with few lines of python which have native JSON support. Please leave only the translation. 
Also, kdb files are not needed, so remove them?